src/App.vue

```diff
@@ -107,7 +107,7 @@ export default {
         params: {
           limit: 200,
           order: 'asc',
-          user: `acct:${this.user}@hypothes.is`,
+          user: `acct:${this.user}${this.user.includes('@') ? '@hypothes.is' : '' }`,
```

Comment:
Ternary reversed, could you also fix that? If not has `@` add hypothes.is. Then all is fine.

Reply:
Whoops! fixed it.